src/display/Display.spec.js

```diff
@@ -1 +1,60 @@
-// Test away!
+
+import React from 'react';
+import { render, fireEvent } from 'react-testing-library';
+import renderer from 'react-test-renderer';
+import 'jest-dom/extend-expect';
```

Comment:
perfect choice since you can use toHaveClass and toBeDisabled for some of the cases in this app